.github/workflows/release.yml

```diff
@@ -117,7 +117,7 @@ jobs:
             build-profile: production
             package: frequency-runtime
             runtime-dir: runtime/frequency
-            built-wasm-file-name-prefix: frequency_rococo_runtime
+            built-wasm-file-name-prefix: frequency_runtime
```

Comment:
For anyone reading this, basically in our releases we should be releasing rococo/mainnet wasms with their respective feature flags on such as ```frequency-rococo-testnet``` ```frequency``` 
To be compatible with how we generate our binaries